### PR TITLE
Document dependencies on 3rd party jars

### LINKED ISSUE #372: Automatically handle third-party libraries
@ph1lb4 - could you repost your emailed question re: translating dependent jars here?

Please provide the relevant j2objcConfig, the version of the plugin you are using, your dependencies { } block if possible, and any errors.

### PR DESCRIPTION
**Pull request by advayDev1**
Documentation for #372 

Fixes #372

### CODE REVIEW

**Comment by advayDev1** [comment]
@brunobowden @ph1lb4 @jjroloff - please take a look

File: FAQ.md [hunk 1]
```diff
@@ -147,12 +147,54 @@ j2objcConfig {
...
+    ...
+}
+```
+
+In the future, this kind of dependency should be inferred automatically from the corresponding
+Java dependency - [issue 41](https://github.com/j2objc-contrib/j2objc-gradle/issues/41).
+
+Also note that this directly incorporates only the neccessary files from your dependencies into
+your Objective C libraries.  It does not produce a separate `gson` Objective C library that you can
+use standalone.  You would need to create a separate Gradle Java project manually to do that.
```
Comment (brunobowden) [inline FAQ.md]: "separate Gradle Java project" - was there an FAQ item about this? If so, then link to it.

**Comment by brunobowden** [comment]
LGTM

File: FAQ.md (hunk 1 above)
Comment (advayDev1) [inline FAQ.md]: nah I haven't this yet.  #419

**Pull request merged**